services/web/server/src/simcore_service_webserver/socketio/server.py

```diff
@@ -36,11 +37,14 @@ def setup_socketio_server(app: web.Application) -> AsyncServer:
         # SEE https://github.com/miguelgrinberg/python-socketio/blob/v4.6.1/docs/server.rst#aiohttp
         # TODO: ujson to speed up?
         # TODO: client_manager= to socketio.AsyncRedisManager/AsyncAioPikaManager for horizontal scaling (shared sessions)
-
+        server_manager = AsyncAioPikaManager(
+            url=get_rabbitmq_settings(app).dsn, logger=_logger
+        )
```

Comment:
Have you checked the `write_only` option to see if it behaves better with multiple instances?

Reply:
this has nothing to do. Please re-read the PR description.
This Manager only allows different server of socketio to communicate among them. The problem comes partialy from the DB notifications mechanism and the fact that we need to update the projects table.

Reply:
Yes, I remember the scenario now! Recent through the issue.